js/base/Exchange.js

```diff
@@ -859,6 +859,50 @@ module.exports = class Exchange {
         };
     }
 
+    currencyStructure () {
+        return {
+            'info': undefined,
+            'id': undefined,
+            'lowercaseId': undefined,
+            'uppercaseId': undefined,
+            'numericId': undefined,
+            'code': undefined,
+            'precision': undefined,
+            'type': undefined,
+            'margin': undefined,
+            'name': undefined,
+            'active': undefined,
+            'deposit': undefined,
+            'withdraw': undefined,
+            'fee': undefined,
+            'fees': undefined,
+            'networks': undefined,
+            'address': undefined,
+            'limits': {
+                'amount': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'price': {
+                    'min': undefined,
+                    'max': undefined,
+                },
+                'cost': {
+                    'min': undefined,
+                    'max': undefined,
+                },
```

Comment:
`amount/price/cost` fields in limits are probably only for symbol. specifically for currency, what does those fields express?

Reply:
I still think these 3 fields are not for currencies, but for markets. therefore we need to remove them from "unified currency structure".